runners/core-construction-java/src/main/java/org/apache/beam/runners/core/construction/WindowingStrategyTranslation.java

```diff
@@ -153,9 +160,13 @@ public static TimestampCombiner timestampCombinerFromProto(RunnerApi.OutputTime
     }
   }
 
-  // This URN says that the WindowFn is just a UDF blob the indicated SDK understands
+  public static final String GLOBAL_WINDOWS_FN = "beam:windowfn:global_windows:v0.1";
+  public static final String FIXED_WINDOWS_FN = "beam:windowfn:fixed_windows:v0.1";
+  public static final String SLIDING_WINDOWS_FN = "beam:windowfn:sliding_windows:v0.1";
+  public static final String SESSION_WINDOWS_FN = "beam:windowfn:session_windows:v0.1";
+  // This URN says that the WindowFn is just a UDF blob the Java SDK understands
   // TODO: standardize such things
-  public static final String CUSTOM_WINDOWFN_URN = "urn:beam:windowfn:javasdk:0.1";
```

Comment:
FWIW the `urn:` at the beginning is the URI scheme that makes it a URN. The official syntax is `"urn:"` _Namespace_ `:` _Namespace-specific format_

(https://tools.ietf.org/html/rfc8141#section-2)

 I agree the scheme is mostly pointless but wanted to just note the departure.